stdpopsim/cli.py

```diff
@@ -625,22 +694,49 @@ def run_simulation(args):
             f"Running simulation model {model.id} for {species.id} on "
             f"{contig} with {len(samples)} samples using {engine.id}."
         )
-        dfe_interval = args.dfe_interval
-        dfe = args.dfe
-        if dfe_interval is None:
-            dfe_interval = ",".join(
-                map(str, [0, int(contig.recombination_map.sequence_length)])
-            )
-        if dfe is not None:
+        # DFE assignment
+        dfe = None
+        dfe_summary_str = None
+        if args.dfe is None:
+            if args.dfe_interval is not None:
+                exit(
+                    "A DFE interval has been assigned without a DFE. "
```

Comment:
I like that you send informative messages, we did not do that. We need to make sure we test these errors.